bin/test_unit

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
```

Comment:
I think it's time we isolated all of the testing stuff in it's own directory.  Take a look at daq and we should roughly mimic daq/testing -- and this script (along with other test_ scripts) should go there.

Reply:
Moved all test_* scripts in travis.yaml to testing